tests/playwright/specs/wikipediaHomePage.js

```diff
@@ -1,14 +1,22 @@
+const { saveVideo } = require( 'playwright-video' );
+const path = require( 'path' ),
+	LOGPATH = process.env.LOG_DIR || path.resolve( __dirname, '..', 'log' );
+
+let capture;
```

Comment:
lets move this declaration inside the `it` block as well

Reply:
This has been changed.